python/lang/security/deserialization/avoid-unsafe-ruamel.py

```diff
@@ -0,0 +1,16 @@
+from ruamel.yaml import YAML
+
+#ok:avoid-unsafe-ruamel
```

Comment:
I'm planning on adding true negative support to `semgrep --test` shortly, but it's not in yet.